src/Engine/Graphics/OpenGL/RenderOpenGL.cpp

```diff
@@ -4594,23 +4594,26 @@ void RenderOpenGL::DrawBuildingsD3D() {
     if (!decal_builder->bloodsplat_container->uNumBloodsplats) return;
 
     for (BSPModel &model : pOutdoor->pBModels) {
-        bool reachable;
-        if (!IsBModelVisible(&model, 256, &reachable)) {
```

Comment:
Why remove this?

Reply:
Removed so blood splats would be applied to models not currently visible to camera.